source/cursorManager.py

```diff
@@ -174,21 +175,41 @@ def run():
 	# Translators: Input help message for NVDA's find command.
 	script_find.__doc__ = _("find a text string from the current cursor position")
 
+	@script(
+		description=_(
+			# Translators: Input help message for find next command.
+			"find the next occurrence of the previously entered text string from the current cursor's position"
+		),
+		gesture="kb:NVDA+f3",
+		resumeSayAllMode=sayAllHandler.CURSOR_CARET,
+	)
 	def script_findNext(self,gesture):
 		if not self._lastFindText:
 			self.script_find(gesture)
 			return
-		self.doFindText(self._lastFindText, caseSensitive = self._lastCaseSensitivity)
-	# Translators: Input help message for find next command.
-	script_findNext.__doc__ = _("find the next occurrence of the previously entered text string from the current cursor's position")
-
+		self.doFindText(
+			self._lastFindText,
+			caseSensitive=self._lastCaseSensitivity,
+			willSayAllResume=willSayAllResume(gesture),
+		)
+
+	@script(
+		description=_(
+			"find the previous occurrence of the previously entered text string from the current cursor's position"

```

Comment:
This message is missing translator commend hence the build failure. For a discussion about making such problems more obvious see #10928